feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/presentation/welcome/PushWelcomeViewModel.kt

```diff
@@ -38,6 +38,7 @@ class PushWelcomeViewModel(
     }
 
     fun backClicked() {
+        welcomePushNotificationsInteractor.setWelcomeScreenShown()
```

Comment:
What about terms and privacy links? As I see it is still ` // Need to implement`

Reply:
We are waiting for Anton to prepare it